##[pylyzer] failed /home/vizn3r/dev/dano-picovina/pdf/venv/lib/python3.12/site-packages/pdfminer/psexceptions.py 1748113092 208
.PSException: ClassType
.PSException <: global::Exception

.PSEOF: ClassType
pdfminer = pyimport "pdfminer"
.PSEOF <: pdfminer.psexceptions.PSException

.PSSyntaxError: ClassType
.PSSyntaxError <: pdfminer.psexceptions.PSException

.PSTypeError: ClassType
.PSTypeError <: pdfminer.psexceptions.PSException

.PSValueError: ClassType
.PSValueError <: pdfminer.psexceptions.PSException

